setup.py

```diff
@@ -14,7 +14,7 @@
     name='centml',
     version='0.1.0',
     packages=find_packages(),
-    scripts=['bin/centml'],
-	install_requires=REQUIRES
+    scripts=['bin/ccompute'],
```

Comment:
can you rebase the code to latest from master?

I believe these changes are not part of your PR